cg/services/order_validation_service/rules/case_sample/utils.py

```diff
@@ -294,3 +296,17 @@ def is_invalid_plate_well_format(sample: Sample) -> bool:
     if sample.is_on_plate:
         return not bool(re.match(correct_well_position_pattern, sample.well_position))
     return False
+
+
+def is_tube_container_name_redundant(sample: Sample, counter: Counter) -> bool:
```

Comment:
Is "redundant" the word we are looking for here? It seems to return true if the tube container name is reused within a case. Maybe "is_tube_name_reused" is more descriptive?